test/commons/color/get-text-shadow-colors.js

```diff
@@ -192,4 +192,154 @@ describe('axe.commons.color.getTextShadowColors', function () {
     assert.equal(shadowColors[0].blue, 0);
     assert.equal(shadowColors[0].alpha, 0);
   });
+
+  describe('with shadows that combine to a stroke', () => {
+    const opt = { minRatio: 0.01 };
+    it('combines multiple shadows without blur into a single stroke', () => {
+      fixture.innerHTML = `
+        <span style="text-shadow:
+          0 -2px #F00,
+          2px 0 #F00,
+          0 2px #F00,
+          -2px 0 #F00;
+        ">Hello wold</span>
```

Comment:
nit for all of these tests:

```suggestion
        ">Hello world</span>
```